### ISSUE TITLE
Custom themes in Scribe

### ISSUE THREAD

**Issue by andrewtavis** [scribe-org/Scribe-iOS#39]
This issue has very low priority. A discussion of how much value custom themes could add to Scribe should eventually be had, with an implementation adding a feature to the eventual settings of the app where users could customize the look of their Scribe keyboards. What would be allowed to be customized and to what degree would be discussed, as well as the way that options would be presented to the users.

- This issue is blocked by #16 as users cannot change their theme without there being settings in the app

**Comment by andrewtavis** [comment]
Closing this issue as the consensus is that for now it's good that Scribe look as much like the system keyboard as possible. Other elements are brand specific and should also not be changed.
